### PR TITLE
doc: add an easy way to test the current development version

### PR DESCRIPTION
**Pull request by sxd**
Signed-off-by: Jonathan Gonzalez V <jonathan.gonzalez@enterprisedb.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,25 @@ CloudNativePG was originally built and sponsored by [EDB](https://www.enterprise
...
+
+```shell
+make deploy-locally
+```
+
+This will create a [KinD](https://kind.sigs.k8s.io/) cluster in your laptop.
+It requires to have [KinD](https://kind.sigs.k8s.io/) and all the dependencies
+installed and also the Go installed to compile the code.
+
+The created cluster can be destroyed like ts:
```
Comment (phisco) [inline README.md:34]:
```suggestion
The created cluster can be destroyed as follows:
```

**Review by phisco — APPROVED** [review]

**Comment by phisco** [comment]
Closes #96

**Review by phisco — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -218,3 +222,11 @@ echo "Downloading $(2)" ;\
 GOBIN=$(PROJECT_DIR)/bin go install $(2) ;\
 }
 endef
+
+kind-cluster: ## Create KinD cluster to run operator locally
+	set -e ;\
```
Comment (mnencia) [inline Makefile:227]: It is the only command. I'm sure the `set -e` is superfluos.

**Comment by gbartolini** [comment]
Please do not merge, as I'd like to review this.

**Review by gbartolini — APPROVED** [review]

**Pull request merged**